### PR TITLE
Crabs and lobstrosities now pack a boatload of puns.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Whale whale whale, It wasn't shrimple, I had to spend severals minutes swimming around to find sea-related punss to use for this PR. I'm otterly and clampletely exhausted.

Soooo, exactly what it reads on the title, however it also applies to the lobster foam helmet, which is contraband from the Autrodrobe.

## Why It's Good For The Game
I thought it'd been interesting to shove a bunch of puns into crustacean mobs. To make them a little bit more unique, and see how many puns I could find or come up with.

## Changelog

:cl:
add: Crabs and lobstrosities (as well as the lobster foam helmet and the fishing hat) now pack a boatload of fish puns.
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
fun fact: for a while i was going to give the lobsters a speech filter like the ones from Stephen King's "The Dark Tower" but then I didn't because I decided it would be kind of a waste of time

**Comment by Sadboysuss** [comment]
in that case, put crabs and lobstrosities in the priority list for the sentient animals list

**Comment by Ghommie** [comment]
> in that case, put crabs and lobstrosities in the priority list for the sentient animals list

Done.

File: strings/crustacean_replacement.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		"kill": [
+			"krill",
+			"gill"
+		],
+		"fanta": [
+			"manta",
+			"finta"
+		],
+		"billion": [
+			"billion",
```
Comment (carlarctg) [inline strings/crustacean_replacement.json:110]:
```suggestion
			"bill-ion",
```

File: strings/crustacean_replacement.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
+
+    "crustacean": {
+		"simp": "shrimp",
```
Comment (ZeWaka) [inline strings/crustacean_replacement.json:4]:
```suggestion
	"crustacean": {
		"simp": "shrimp",
```
consistent formatting

**Comment by EricZilla** [comment]
Its shrimple, we krill the batmanta

If you are accepting puns
"Doofus" to "Doofish"
"Could" to "Cod"
"Hell of it" to "Halibut"
"What are" to "Water"
"Other" to "Otter"
"Lots" to "Lobst"
"Fumble" to "Flounder"
"Serious" to "Searious"
"Show" to "Shoal"
"Miracle" to "Mackerel"
"Trait" to "Trout"
"Syndicate" to "Sea World"
"Syndi " (there is a space here) to "Fisherman "
"Spy" to "Marine Biologist"
"Heretic" to "Herringtic"
"Cult" to "Shoalt"
"Nar'Sie" to "Carp'Sie"
"Blob" to "Blobfish"
"Dumbass" to "Dum Bass"

I could keep going but I should not

**Comment by EricZilla** [comment]
ACTUALLY FOR REAL CAN THE FISHING HAT DO THIS TOO?

File: strings/crustacean_replacement.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+		"below": "downstream",
+		"heretic": "herringtic",
+		"cult": "shoalt",
+		"nar'si": "carp'si",
+		"narsi": "carpsi",
+		"blob": "blobfish",
+		"dumbass": "dum bass",
+		"outdone": "troutdone",
+		"quintessen": "squiddessen",
+		"clos": "claws",
```
Comment (Jacquerel) [inline strings/crustacean_replacement.json:131]: Is this meant to be "close"? I don't know what "clos" is

Comment (Ghommie) [reply strings/crustacean_replacement.json:131]: Yes, it should fit on words like close, closet, closing and closure: clawse, clawset, clawsing, clawsure.

**Comment by deathrobotpunch** [comment]
> You're going to shell for this.

cod you please calm down? you’re being a reel pain in the bass

**Comment by datorangebottle** [comment]
> > You're going to shell for this.
> 
> cod you please calm down? you’re being a reel pain in the bass

Shuck you, man.

**Comment by Ghommie** [comment]
> _Shuck_ you, man.

Uh, I gotta add that one.

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**